documentation/applications/index.markdown

```diff
@@ -0,0 +1,12 @@
+---
+author: yuyan
+comments: false
+date: 2019-4-29 18:00:00+00:00
+layout: page
+slug: code
+title: Applications
+---
+
+# MoveIt Applications
+
+There are many diverse application examples of what you can use MoveIt for. The list below demonstrates some of the advanced applications developed on MoveIt. You can have fun by trying the applications, which may help you figure out how to fit different features together. If you are interested, you can visit [MoveIt Tutorials](https://ros-planning.github.io/moveit_tutorials/) for further technical details. At the same time, your contribution is encouraged, no matter the application is developed for some usages, competitions or research topics, or to demonstrate the newly developed MoveIt feature.
```

Comment:
Please add https://github.com/ros-planning/moveit_example_apps

Reply:
Added a small section introducing `moveit_example_apps` and a list holding large open source projects using MoveIt.